src/utils/hitConverter.js

```diff
@@ -183,8 +182,13 @@ class HitConverter{
             }
         } else {
             // no mappings, iterate setting each field directly (less overhead)
-            for(var i=0; i<indexConfig.returnFields.length; i++){
-                feature.properties[indexConfig.returnFields[i]] = hit._source[indexConfig.returnFields[i]];
+            // return whatever is in the hits source.  By default this will be configured fields
+            // but if outFields was set it can be overridden.
+            let hitFields = Object.keys(hit._source);
+            for(let i=0; i<hitFields.length; i++){
```

Comment:
Are the number of keys in hitFields likely to be very large?  If this `for` loop doesn't need to iterate a huge amount of times, you're better off using `map` for readability.  Can you paste in an example of the contents of `hitFields`?

Reply:
It completely depends on the index in elasticsearch. It could possibly be 1k plus, so just to be safe I use a for loop.

Reply:
Ok, let's leave the `for`s.  At some point we should refactor this file so that the logic and code is moved into smaller named functions.